### DISCUSSION TITLE
How is the size of the binary compared to AppImage?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [flatimage/flatimage#3] [General]
Is it greater, is it less, or is it the same?
it would be nice to put examples and comparisons (in size, time to open...) in the readme :+1:

**Comment by ruanformigoni** [comment]
Hi @Heus-Sueh , thanks for your feedback! It is usually larger, since arts is portable and cannot use host libraries, appimage by default mounts a ro filesystem and runs a program in it without any form of isolation (similar to pluging in a flash drive and executing a program in it); workarounds for this are `AppRun` and `appimagebuilder`, however they can't cover everything (like glibc in wine) and the program still runs on the host's root. I'll update the README.md with the mentioned info :)
